test/main.js

```diff
@@ -61,50 +47,39 @@ before('initialization', function (done) {
 		chatRoomsData.loaded = true;
 	} catch (e) {}
 
-	// Actually crash if we crash
-	config.crashguard = false;
-
-	// Don't try to write to file system
-	config.logladderip = false;
-	config.logchallenges = false;
-	config.logchat = false;
-
 	// Don't create a REPL
 	require('./../repl').start = noop;
 
+	// `watchFile` is unsupported and throws with mock-fs
+	Object.defineProperty(fs, 'watchFile', {
+		get: function () {return noop;},
+		set: noop,
+	});
+
 	// Sandbox file system: it's possible for a production server to be running in the same directory.
-	// And using a sandbox is safer anyway.
-	const fsSandbox = {
+	// And using a sandbox is safer anyway..
+	mock({
 		'config': {},
 		'chat-plugins': getDirTypedContentsSync('chat-plugins', 'file'),
 		'mods': getDirTypedContentsSync('mods', 'dir'),
 		'logs': {
 			'chat': {}, 'ladderip': {}, 'modlog': {}, 'repl': {},
 			'lastbattle.txt': '0',
 		},
-	};
-
-	// `watchFile` is unsupported and throws with mock-fs
-	Object.defineProperty(fs, 'watchFile', {
-		get: function () {return noop;},
-		set: noop,
 	});
-	mock(fsSandbox);
 
-	init(done);
-});
+	// Finish initializing.
```

Comment:
Nothing in particular. I put it there as a reminder of what ``init`` was originally doing while I was working on this and never took it out